src/screens/Messages/Conversation/MessagesList.tsx

```diff
@@ -223,13 +234,10 @@ export function MessagesList() {
   }, [isMomentumScrolling])
 
   const scrollToEnd = React.useCallback(() => {
-    requestAnimationFrame(() => {
-      if (isMomentumScrolling.value) return
-
-      flatListRef.current?.scrollToEnd({animated: true})
-      isMomentumScrolling.value = true
-    })
-  }, [isMomentumScrolling])
```

Comment:
Since this is happening sync now, we don't need to use `requestAnimationFrame` here.